sdk/docs/iot/pnp.md

```diff
@@ -0,0 +1,27 @@
+# Azure IoT Plug and Play Client
+
+Official Embedded C client for Azure IoT Plug and Play.
+
+## Getting Started
+
+- **Feature**: If you are new to IoT Plug and Play, take a look at the [Intro for Plug and Play here](https://docs.microsoft.com/en-us/azure/iot-pnp/overview-iot-plug-and-play).
```

Comment:
Link verification is failing because of `en-us`.

```suggestion
- **Feature**: If you are new to IoT Plug and Play, take a look at the [Intro for Plug and Play here](https://docs.microsoft.com/azure/iot-pnp/overview-iot-plug-and-play).
```